src/compiler/diagnosticMessages.json

```diff
@@ -5733,5 +5737,9 @@
     "An optional chain cannot contain private identifiers.": {
         "category": "Error",
         "code": 18030
+    },
+    "Checking this statement may result in the creation of as many as {0} types and {1} symbols.": {
+        "category": "Warning",

```

Comment:
This is the first `Warning`.  It feels pretty distinct to me, but it doesn't really matter one way or the other.